packages/buildcop/src/buildcop.ts

```diff
@@ -510,7 +510,16 @@ buildcop.markIssueFlaky = async (
   const existingLabels = existingIssue.labels
     ?.map(l => l.name)
     .filter(l => !l.startsWith('buildcop'));
-  const labels = LABELS_FOR_FLAKY_ISSUE.concat(existingLabels);
+  let labels = LABELS_FOR_FLAKY_ISSUE;
```

Comment:
Up above is a linter warning.  I wanted to make warnings errors but forgot.  Mind dropping `title` to keep linter errors down?

Reply:
Done.